MNE/disp3D/3DObjects/brain/brainrtconnectivitydatatreeitem.cpp

```diff
@@ -109,6 +109,16 @@ BrainRTConnectivityDataTreeItem::BrainRTConnectivityDataTreeItem(int iType, cons
 
 BrainRTConnectivityDataTreeItem::~BrainRTConnectivityDataTreeItem()
 {
+    setVisible(false);
+
+    delete m_pRenderable3DEntity;
+    delete m_pItemNetworkThreshold;
+
+    for(int i = 0; i < m_lNodes.size(); ++i) {
+        delete m_lNodes[i];
+    }
+
+    m_lNodes.clear();
```

Comment:
are we delaing with QOBjects here? - Consider using QPointer instead of Raw Pointer -> destruction would be handled automatically